make ECS8's SelectProfile a little nicer

quietly-turning opened the pull request:
These commits add support for displaying each player's national flag (if their country was specified  on GrooveStats), avatar (if provided via GrooveStats), and SRPG3 Level on ScreenSelectProfile.

<img width="2032" alt="Screen Shot 2019-09-06 at 10 44 14 AM" src="https://user-images.githubusercontent.com/1253483/64437439-4998d200-d094-11e9-8a25-5ba7fbb53e90.png">

It's not critical to the theme's functioning, but I think it feels better. 🙂 

I tried to break the work into sensible commit chunks so you could keep some and revert others if you're inclined.

quietly-turning committed:
tidy spacing on ECS.Players table in ./Scripts/ECS

quietly-turning committed:
convert remaining tabs to spaces in ECS.lua

Quickly, before anyone from either side of the tabs vs spaces war notices that I was briefly using both in a single file.

quietly-turning committed:
add country, level, and exp to ECS.Players

I obtained this data from http://groovestats.com/SRPG3/index.php?page=ranking at 7am Eastern Time, so it's as up to date as it can be.

Not every player in the ECS.Players Lua table had a corresponding row at the GrooveStats SRPG3 page.  In those cases, I made country an empty string, and level and exp I set as nil.

Nothing in the theme uses this yet.  Hopefully soon.

quietly-turning committed:
fix mixup in ECS.Players

I had accidentally mixed up the countries of AV6 and Pluto.

quietly-turning committed:
make ECS8 SelectProfile a little nicer

Country flag graphics obtained from https://emojipedia.org/flags/

Player avatars acquired from http://groovestats.com/SRPG3/i (and YouTube, in the case of Bran).

Some avatars were jpg, and those were having strange zoom issues (probably ActorProxy related) so I converted all avatars to be png.

teejusb commented:
I also JUST added GALAGA. Is she part of this list?

Also are we using spaces instead of tabs now? I think a lot of the other code in SL is tab separated so I mostly tried to keep it consistent but /shrug.

quietly-turning commented:
SL uses tabs 99% of the time, yeah.
Galaga was not in ECS.lua when I was working on this.  Give me a moment to fix that.

quietly-turning committed:
Merge branch 'ECS8' of https://github.com/teejusb/Simply-Love-SM5 into ECS8-pr1

quietly-turning committed:
add some missing data from ECS.Players

This adds country, level, and exp values for:
  Koettmaskinen
  Yung Kiki
  I_ONLY_PLAY_BOARD_GAMES_NOW
  Sal!V2
  HOT TAKE

It also adds assets for the Swedish national flag and GALAGA's avatar.

quietly-turning committed:
spaces -> tabs for ECS.lua
